lib/metasploit/framework/data_service/proxy/session_event_data_proxy.rb

```diff
@@ -1,19 +1,19 @@
 module SessionEventDataProxy
 
-  def session_events
-    begin
-      data_service = self.get_data_service()
-    rescue  Exception => e
-      elog "Call to  #{data_service.class}#session_events threw exception: #{e.message}"
-    end
-  end
+  # def session_events
```

Comment:
Why did this method get commented out?

Reply:
not used

Reply:
Any reason to hang on to it?

Reply:
Aren't we planning on making stuff like session events available at some point for history tracking and such (thinking in the frontend product). Should we just dump this now or hold on to it and update it when we get to that stage?

Reply:
I'll dump it since its useless